### ISSUE TITLE
How to train my model?

### ISSUE THREAD

**Issue by aboy2018** [tensorflow/models#7213]
I train my classifier model that uses "mobilenetv1_model",but my classifier num is two,what I can do? The wrong problem is 

"InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [1,1,1024,2] rhs shape= [1,1,1024,10]
	 [[Node: save/Assign_301 = Assign[T=DT_FLOAT, _class=["loc:@MobilenetV1/Logits/Conv2d_1c_1x1/weights"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](MobilenetV1/Logits/Conv2d_1c_1x1/weights/RMSProp, save/RestoreV2:301)]]
"
Thank you!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by aboy2018** [comment]
> Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
> What is the top-level directory of the model you are using
> Have I written custom code
> OS Platform and Distribution
> TensorFlow installed from
> TensorFlow version
> Bazel version
> CUDA/cuDNN version
> GPU model and memory
> Exact command to reproduce

Thanks for you reply. I don't use pred-model, the training path is "/home4/jiashuaihe/projects/models-master/research/slim/train_image_classifier.py" and my environment is that "linux+python3.6+tensorflow-gpu1.12.0+cuda9.2"
I noticed that there be three classes(flowers,mnist,cifar10) .
